policy/blacklist.go

```diff
@@ -1,1027 +0,0 @@
-// Copyright 2014 ISRG.  All rights reserved
```

Comment:
Please include a sample black-list in the repo, especially for things like the special use domains.